morpheus/pipeline/receiver.py

```diff
@@ -71,58 +72,122 @@ def is_partial(self):
 
     @property
     def in_pair(self):
-        return (self.in_stream, self.in_pair)
+        return (self.in_node, self.in_pair)
 
     @property
-    def in_stream(self):
-        return self._input_stream
+    def in_node(self):
+        return self._input_node
```

Comment:
Important: Delete this property. It's only being used by `in_pair`, which should also be deleted.